src/styles/mixins/buttons.scss

```diff
@@ -8,3 +8,59 @@
   background-color: transparent;
   border-style: none;
 }
+
+/**
+ * A <button> composed of an SVG icon (left) and text (right) with some
+ * hover transition effects
+ */
+@mixin action-button {
```

Comment:
Are the various uses of `action-button` similar enough that we could extract a Preact component for them?

Reply:
Possibly. Bears discussion...I feel like the jury is still out and that right now would be premature. Happy to discuss further.